src/main/java/org/zalando/nakadi/validation/EventTypeOptionsValidator.java

```diff
@@ -28,11 +27,11 @@ public void validate(final Object target, final Errors errors) {
     }
 
     private void checkRetentionTime(final Errors errors, final EventTypeOptions options) {
-        if (Objects.nonNull(options) && Objects.nonNull(options.getRetentionTime())) {
-            final Long retentionTime = options.getRetentionTime();
-            if (retentionTime > maxTopicRetentionMs) {
+        final Long retentionTime = options.getRetentionTime();
+        if (retentionTime != null) {
+            if (retentionTime.longValue() > maxTopicRetentionMs) {
```

Comment:
But for me it seems strange that you don't use autounboxing but use autoboxing, here for example: https://github.com/zalando/nakadi/pull/365/files#diff-ae87fdc01827028b148be3085e54d23eR85
I'm not aware of any performance issues with auto- boxing/unboxing I think it's only about the coding style.
I think we use auto boxing/unboxing elsewhere in the project, so I don't really see much reasons to use manual unboxing in this single place.